packages/vendor-core/package.json

```diff
@@ -66,15 +67,14 @@
     "react-router-bootstrap": "^0.24.4",
     "react-router-dom": "^4.3.1",
     "redux": "^3.6.0",
-    "redux-form": "8.2.0",
-    "redux-form-validators": "^2.1.2",
```

Comment:
Do we actually want to drop the support for those libraries? Isn't it going to break core/plugins?

Since this commit contains `breaking changes` please run `npm run commit` and answer the `breaking changes` question with `drop support for redux-form and redux-form-validator`.

Reply:
It should not break plugins as they are not using redux-form, at least I did not find such plugin. For core, I have https://github.com/theforeman/foreman/pull/6832

But maybe we could do it in 2 steps - add new packages first and when changes in core are merged, remove the redux-form and related to have a smooth transition. What do you think?

Reply:
+1 IMO if we can safely remove `redux-form` while we replace it with `formik` let's do it :+1: 

Would you like me to release a `@theforeman/vendor` version `1.0.0-formik.0`? Will it help you and other developers to test your PR?

Reply:
Having a new release with the changes would be great, that way we would also see if the tests on CI are passing.

Reply:
You can use in your PR:
- `@theforeman/vendor` version `1.0.0-formik.0`:
https://www.npmjs.com/package/@theforeman/vendor/v/1.0.0-formik.0
- `@theforeman/vendor-dev` version `1.0.0-formik.0`:
https://www.npmjs.com/package/@theforeman/vendor-dev/v/1.0.0-formik.0